sklearn/ensemble/_hist_gradient_boosting/_predictor.pyx

```diff
@@ -20,8 +21,9 @@ def _predict_from_numeric_data(nodes, numeric_data, out):
     _predict_from_numeric_data_parallel(nodes, numeric_data, out)
 
 
-def _predict_from_binned_data(nodes, binned_data, out):
-    _predict_from_binned_data_parallel(nodes, binned_data, out)
+def _predict_from_binned_data(nodes, binned_data, support_missing_values, out):
+    _predict_from_binned_data_parallel(nodes, binned_data,
```

Comment:
didn't you say you don't need the different functions for the parallel to work?

Reply:
We do here, the parallel functions are `cdef`ed

During the GBDT review it was suggested to have `predictor.py` and `predictor.pyx`, separating cython and pure python code. That means we need these kind of wrappers.

Reply:
@adrinjalali Did you mean something like this? https://github.com/scikit-learn/scikit-learn/pull/14003

Reply:
@thomasjpfan precisely!

@NicolasHug I thought if we have a `cpdef` function in `.pyx` files, they can handle the usecase being only one function. Or do we actually need this to be in a `.py` file?

(really no big deal though)

Reply:
Yes I was wrong, we can use `prange` in a `def` function. We can merge #14003 proposed by Thomas.